src/components/UsernameForm/TimeMessage/getTimeMessage.js

```diff
@@ -1,15 +1,19 @@
+import dayjs from 'dayjs';
```

Comment:
@bhattaraib58 A new library to just get difference in days? Oh man, we should create our own util and use that instead.

We shouldn't be adding new packages for simple stuff. I know dayjs is only 2kb and might also support tree shaking.

Reply:
Nevermind. This project doesn't need to be follow all the rules.